### DISCUSSION TITLE
low-pass filter only applied on music

### DISCUSSION THREAD

**Discussion by integerrr** [ppy/osu#15503] [Q&A] [answered]
Not sure if this is intentional, but the low-pass filter doesnt apply on sounds like hitsounds, probably isnt a big issue but I can foresee it might be jarring if features like the low-pass filter being applied when game is unfocused gets implemented, and someone watching a replay alt+tab out of the game, they might have this issue. 

https://user-images.githubusercontent.com/7279624/140625551-176171c7-2d32-4951-a643-b8cb7598b5e4.mp4

**Comment by peppy** [comment, marked as answer]
It is intended
